src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="http://localhost:3000/api"
```

Comment:
wrong action on submit - On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application